File: c++/bucle_while.cpp
#include <iostream>

using namespace std;

int main() {
    // Bucle while con contador
    int contador = 0;
    while (contador < 10) {
        contador++;
        cout << "Iteración " << contador << endl;
    }

    // Bucle infinito con break
    while (true) {
        int a;
        cout << "Introduzca un valor mayor que 5: ";
        cin >> a;

        if (a > 5) {
            cout << "Es correcto" << endl;
        } else {
            cout << "Es incorrecto, pruebe de nuevo" << endl;
            break;
        }
    }

    return 0;
}

File: Python/bucle_while.py
contador = 0
while contador < 10:
    contador += 1
    print("Iteración", contador)

#While=True combinado con break nos permite un bucle infinito mientras no se cumpla la condición

while True:
    print("Introduzca un valor mayor que 5")
    a = int(input())
    if a > 5:
        print("Es correcto")
       
    else:
        print("Es incorrecto, pruebe de nuevo")
        break